### PR TITLE
Add description for `ORCID iD`

### PR DESCRIPTION
**Pull request by nikosev**
# Summary

This PR adds the description of `ORCID iD` attribute

### CODE REVIEW

**Review by vardizzo-lab — APPROVED** [review]
OK.

File: content/en/providers/check-in/sp/_index.md [hunk 1]
```diff
@@ -1190,6 +1190,23 @@ connected to Check-in.
 |               **notes** | -                                                                                                                                                 |
 |              **status** | Experimental                                                                                                                                      |
 
+### 16. ORCID iD
+
+|          attribute name | ORCID iD                                               |
+| ----------------------: | :----------------------------------------------------- |
+|         **description** | Provides information about the user's ORCID iD         |
```
Comment (gwarf) [inline content/en/providers/check-in/sp/_index.md:1197]:
```suggestion
|         **description** | Provides information about the user's ORCID ID         |
```

File: content/en/providers/check-in/sp/_index.md [hunk 2]
```diff
@@ -1190,6 +1190,23 @@ connected to Check-in.
 |               **notes** | -                                                                                                                                                 |
 |              **status** | Experimental                                                                                                                                      |
 
+### 16. ORCID iD
+
+|          attribute name | ORCID iD                                               |
```
Comment (gwarf) [inline content/en/providers/check-in/sp/_index.md:1195]:
```suggestion
|          attribute name | ORCID ID                                               |
```

**Review by gwarf — COMMENTED** [review]
`iD` is reported by the language linter and looks weird, can we just move to `ID` or is it really incorrect there?

**Comment by gwarf** [comment]
See https://support.orcid.org/hc/en-us/articles/360006897334-What-is-my-ORCID-iD-and-how-should-I-use-it- but it looks really weird...

**Comment by gwarf** [comment]
Even them are not always coherent:
Screenshot 2022-03-18 at 15 36 31

Let me know what's the preference and if needed I could merge bypassing this specific linting error

**Comment by nikosev** [comment]
Hello @gwarf 

I agree with you that `ORCID iD` feels weird, but I think we should follow the format as ORCID promotes it.
So, could you please bypass the linter error and merge the PR?

Have a nice weekend,
Nick

**Pull request merged**